server/routes/request.js

```diff
@@ -0,0 +1,14 @@
+const express = require("express");
+const {
+  getRequests,
+  makeRequest,
+  updateRequest,
+} = require("../controllers/request");
+const protect = require("../middleware/auth");
+const router = express.Router();
+
+router.route("/").get(protect, getRequests);
+router.route("/").post(protect, makeRequest);
+router.route("/").put(protect, updateRequest);
```

Comment:
It's a good practice to pass the id of the resource to be updated in the request params. You can make this one a dynamic route.